pages.tr/common/alias.md

```diff
@@ -1,18 +1,18 @@
 # alias
 
-> Takma adlar/kısayollar oluşturur -- bir komut dizesi ile değiştirilen sözcükler.
+> Takma adlar/kısayollar (bir komut dizesi ile değiştirilen sözcükler) oluşturur.
```

Comment:
Should this also not be in imperative form ("oluştur")?

Reply:
Sure but its OK like that too as this is the description and not an instruction.

Reply:
Ah, all good then, my bad!